prisma/schema.prisma

```diff
@@ -73,10 +73,23 @@ model Universities {
     courses    Courses[]
     professors Professors[]
     faculties  Faculties[]
+    domains    UniversityDomains[]
 
     @@map("universities")
 }
 
+model UniversityDomains {
+    id                   Int          @id @default(autoincrement())
+    domain               String       @unique
+    belongToUniversity   Universities @relation(fields: [belongToUniversityId], references: [id])
+    belongToUniversityId Int          @map("belong_to_university")
+    createdAt            DateTime     @default(now()) @map("created_at") @db.Timestamptz(3)
+    updatedAt            DateTime     @updatedAt @map("updated_at") @db.Timestamptz(3)
+
+    @@unique([domain, belongToUniversityId])
```

Comment:
Question: What's the reason of creating a composite key? Do we expect 2 unis to share the same domain?

Reply:
thanks for the review!

dont expect 2 unis to share same domain, but that check is already accounted for by declaring uniqueness for domains

removed this redundant composite unique keys at [8e81fcd](https://github.com/AfterClass-io/afterclass.io-v2/pull/136/commits/8e81fcd7ffa1450c1ce0919b23adfe6cee0bd640)